script.js

```diff
@@ -36,11 +39,9 @@ function generateQuote() {
     });
   tweetBtn.addEventListener("click", shareTweet);
 
-  shareTweet();
-
```

Comment:
Removes `shareTweet()` call without click, as it triggers a popup which is blocked by browser popup-blockers anyway (eg. likely wont do anything)

Reply:
`shareTweet()` is unnecessary at all because it is called by click. Thanks for pointing out!